### PR TITLE
Added if statement around hpo section so only shows when available in…

### PR DESCRIPTION
**Pull request by jessgilbert**
… active language

Whole HPO section disappears if the pages aren't available in the active language

### CODE REVIEW

**Comment by jessgilbert** [comment]
> could you just do this instead? `if page.high_potential_opportunities|filter_by_active_language` as the filter will return an empty list (which is falsey) if there are none in the requested language

Yes I did have this and it works, but Mike suggested adding a filter which is clearer and returns a boolean so as to work better in this context

**Comment by eadpearce** [comment]
> Mike suggested adding a filter which is clearer and returns a boolean so as to work better in this context

that's fair. it does make it easier to read in the template

**Comment by Kenneth-Siu** [comment]
This sort of thing feels like something we'd want in directory-components?

1) Is there already something like that?
2) If not, maybe it would be worth putting it there?

File: core/templates/core/landing_page.html [hunk 1]
```diff
@@ -86,10 +86,11 @@ <h2 class="heading-large">{{ page.sector_title }}</h2>
     <div class="card-grid padding-bottom-45">
       {% include 'core/includes/card_with_description_grid.html' with url_name='industry' cards=page.sectors|filter_by_active_language  %}
     </div>
-    <a href={{ page.sector_button_url }}  class="button button-arrow-small">{{ page.sector_button_text }}</a>
+    <a href="{{ page.sector_button_url }}" class="button button-arrow-small">{{ page.sector_button_text }}</a>
   </div>
 </section>
 
+{% if page.high_potential_opportunities|are_available_in_active_language %}
```
Comment (richtier) [inline core/templates/core/landing_page.html]: would the existing `{% if page.high_potential_opportunities|filter_by_active_language %}` do the job?

`filter_by_active_language` will return an empty list if not available in the active language, and thus be evaluated as falsey

**Comment by richtier** [comment]
regarding putting it in directory-components: not yet. rule of thumb is:
 - if we use the same code in two services then consider putting it in directory-components
 - if we use the same code in three services then put it in directory-components :)

**Comment by richtier** [comment]
the motivation for the filter is laudable: so it's more readable and maintainable

however, more logic in templates and an additional custom template tags adds a lot of mental overhead to developers compared with performing the check in the view

```
def get_context_data(self, **kwags):
    pages = page.high_potential_opportunities
    return super().get_context_data(
        show_hpo_section=bool(pages and filter_by_active_language(pages)),
        **kwargs
)
```

less moving parts compared with a custom template tag

**Pull request merged**